client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -175,3 +190,12 @@ def get_selection(self):
             )
 
         return selection
+
+
+class createStaticMesh(CreateFBX):
+    """Static Meshes as FBX. """
+
+    identifier = "io.openpype.creators.houdini.staticmesh.fbx"
+    label = "staticMesh (FBX)"
+    product_type = "staticMesh"
+    icon = "cube"
```

Comment:
This should override publish families again - because otherwise it'd still enforce `model` and `fbx` instead of static mesh since it inherits from the other class.
```suggestion
    icon = "cube"

    def get_publish_families(self):
        return ["fbx", "staticMesh"]
```